pkg/volume/empty_dir/empty_dir.go

```diff
@@ -206,7 +207,7 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 	// the container.
 	opts := ed.getTmpfsMountOptions()
 	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
-	return ed.mounter.Mount("tmpfs", dir, "tmpfs", 0, opts)
+	return ed.mounter.Mount("tmpfs", dir, "tmpfs", nil)
```

Comment:
@ddysher If you pass `nil` here, you will drop the `rootcontext` option and this feature will not work when selinux is enforcing.  Needs to be fixed.